derive_builder_core/src/setter.rs

```diff
@@ -210,6 +223,11 @@ impl<'a> ToTokens for Setter<'a> {
     }
 }
 
+/// Returns expression wrapping `bare_value` in `Some`
+fn wrap_expression_in_some(bare_value: TokenStream) -> TokenStream {
```

Comment:
```suggestion
fn wrap_expression_in_some(bare_value: impl ToTokens) -> TokenStream {
```

This would avoid forcing the caller to allocate a token stream unnecessarily.